docs/guides/development/nodejs/nodejs-twitter-bot/index.md

```diff
@@ -0,0 +1,294 @@
+```YAML
+author:
+     name: Pj Metz
+     email: metz.pj@gmail.com
+description: 'Building a new Twitter bot and hosting it on Linode. Guide is for beginners new to coding and cloud services.'
+keywords: ["node.js", "beginner", "javascript", "twitter bot"]
+license: '[CC BY-ND 4.0](https://creativecommons.org/licenses/by-nd/4.0)'
+published: 2021-04-27
+modified_by:
+	name: Linode
+title: 'Node.js Twitter Bot Beginner’s Guide: Replying to Users and Scheduling Tweets'
+contributor:
+	name: Pj Metz
+	link: [Metzinaround.com](https://www.metzinaround.com)
+external_resources:
+	- "[Creating a new User in Ubuntu 		 (https://youtu.be/fDHHKR0nVQg)"
+	- "[FileZilla](https://www.linode.com/docs/guides/filezilla/)
+```
+
+#  Node.js Twitter Bot Beginner’s Guide: Replying to Users and Scheduling Tweets
+
+Twitter bots are a great way to practice using an API and developing some coding skills along the way. You can usually get the bot going for under 100 lines of code, so it’s especially good for beginners. In this tutorial, I’ll show you how to use Node.js, *a framework for javascript* and a few npms, or *node package managers* to have your bot tweeting in no time.
+
+## Before you Begin:
+
+You should download or install:
```

Comment:
I expanded this into a "Install Software" section https://deploy-preview-4492--nostalgic-ptolemy-b01ab8.netlify.app/docs/guides/nodejs-twitter-bot/#install-software-on-your-workstation

Also added a few more bits of software to this section:
- The git command line software; later in the guide I inserted instructions for using Git at the command line as an option (while still also including GitHub Desktop as the other option)
- How to install Node and npm on the user's workstation, via NVM. This is because the user may not have Node/npm on their system to start

Beyond that, I let the user know that VS Code is a recommended editor, but that other plain text editors are ok too